.circleci/config.yml

```diff
@@ -199,7 +199,9 @@ commands:
       - run:
           name: Setup venv
           command: |
-            python3 -m venv /tmp/venv
+            sudo apt-get update
+            sudo apt-get install -y python3-venv
+            /usr/bin/python3 -m venv /tmp/venv
```

Comment:
For context: I don't know why it's not pre-installed in the new images, but `python3 -m venv` will fail and end up creating a Python 2 environment if you don't installed python3-venv first. Also note that this switches us to the system Python (3.8.5 on Ubuntu 20.04) instead of using pyenv. The new images have pyenv pre-install 3.9.x, which causes a lot of problems for our dependencies at this point.